overrides/common/style.ts

```diff
@@ -40,3 +41,36 @@ export const decorativeHeader = css`
     background: ${themeVal("color.primary")};
   }
 `;
+
+export const accessibilityLinks = styled.a`
+  text-decoration: underline;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &:focus {
+    outline: 3px solid ${themeVal("color.primary")};
+  }
+
+  &:active {
+    outline: 3px solid black;
+  }
+`;
+
+export const accessibilityMenuItems = styled(NavLink)`\
+  &:hover {
+    cursor: pointer;
+    text-decoration: underline;
+  }
+
+  &:focus {
+    outline: 3px solid ${themeVal("color.primary")};
+  }
+
+  &:active {
+    outline: 3px solid ${themeVal("color.primary")};
```

Comment:
There seems to be quite overlap between a style and link style (understandably.) would it be beneficial to have them use the same style using the css block like https://github.com/NASA-IMPACT/veda-config-eic/blob/0e1150024a11c6ba435a186e91a902faf5c95755/overrides/common/style.ts#L12 ?